# 传感器感知到的数据
float64 leftWheelVelocity        # 左轮速度
float64 rightWheelVelocity       # 右轮速度
float64 pitch                    # 平衡车俯仰角
float64 yaw                      # 平衡车偏航角
float64 pitchAngVelocity         # 平衡车俯仰角角速度
float64 yawAngVelocity           # 平衡车偏航角角速度
# 键盘控制节点收到的控制消息
float64 dst_vel                  # 期望速度
float64 dst_yaw                  # 期望偏航角
---
# 平衡车两轮的控制力矩
float64 leftTorque
float64 rightTorque